app/src/main/java/net/kuama/documentscanner/utils/PointUtils.kt

```diff
@@ -0,0 +1,30 @@
+package net.kuama.documentscanner.utils
+
+import net.kuama.documentscanner.data.Corners
+import org.opencv.core.Point
+import org.opencv.core.Size
+import java.util.*
+import kotlin.Comparator
+
+object PointUtils {
+    fun getSortedCorners(it: Array<Point>, originalSize: Size): Corners {
+        val cornerList = it.toList()
```

Comment:
if you convert it to a list immediatly, why don't you accept `List<Point>` in the constructor directly? isn't it better? 😄